test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash -xv
+
+source test/init
+
+plan tests 16
+
+source openqa-label-known-issues
+client_args=(api --host "$host_url")
+
+export KEEP_REPORT_FILE=1
+
+client_output=''
+mock-client-output() {
+    client_output+="client_call $@"$'\n'
+    echo "$client_output\n"
+}
+
+client_call=(mock-client-output "${client_call[@]}")
+autoinst_log=$dir/test/data/04-autoinst.txt
```

Comment:
`$dir` is set `in `test/init` to the test directory itself.

Reply:
if you print it is `~/some_path/scripts`. what do you mean?

Reply:
What I mean is, that `test/init` sets `dir=...`.
And it is set to the test directory itself. So `/some/path/scripts/test`.

You don't need to define `dir=...` in `openqa-label-known-issues`.
Remove that definition and then you can use
```suggestion
autoinst_log=$dir/data/04-autoinst.txt
```

Reply:
00a7075